libs/application/templates/parental-leave/src/lib/answerValidators.ts

```diff
@@ -178,7 +179,14 @@ export const answerValidators: Record<string, AnswerValidator> = {
     }
     return undefined
   },
-  // TODO: should we add validation for otherParent's email?
+  [OTHER_PARENT_EMAIL]: (newAnswer: unknown, application: Application) => {
+    const email = newAnswer as string
+    const { otherParent } = getApplicationAnswers(application.answers)
+    if (otherParent !== NO && otherParent !== SINGLE && !isValidEmail(email)) {
```

Comment:
I would consider reversing this boolean for clarity:

`if (otherParent === YES && otherParent !== SINGLE && !isValidEmail(email)) { ... }`